.circleci/config.yml

```diff
@@ -858,28 +858,28 @@ jobs:
             mkdir -p ~/artifacts/MELA/
             mv *.out ~/artifacts/MELA/
           when: on_fail
-      - run:
-          name: "testME_more"
-          command: cd test; ./testME_more.py "$(expr $SEED + $CIRCLE_BUILD_NUM)"
-          when: always
```

Comment:
Is there a way to keep this check?

Reply:
I do not think so at this point. The reason we cannot do this is because the docker image we use to run CircleCi does not have the modules required for the python3 binding. Another complication is that we do not have root access in the docker image, so we cannot install the py bindings. The other docker images of CMSSW did not work either since they all  do not have python3 installed either. I also worked on trying to use the cmssw/cc7 enviornment in CircleCi, but again we need to somehow mount cvmfs and use singularity on the circleCi nodes. For now, since we independently did these ME checks for this PR. I suggest we just make sure that whoever pushes to JHUGen runs this script manually. In the meantime we will look for a way to keep this check.

Reply:
@usarica does this seem reasonable?

Reply:
One thing I notice is that the commit id of MELA is not up-to-date; in fact, this test file was updated by Mohit to account for correct bindings. Would it be possible to try with the head of the current JHUGenMELA master to see if that helps resolve the issues?

In addition, would you mind listing here the docker images you used for testing?

Reply:
Yeah I can do that. 

Here are the images, where we would not have to mount cvmfs that I tried (in addition to the version we use now):

cmssw/cmssw:CMSSW_11_1_3
cmssw/cmssw:CMSSW_11_0_3
cmssw/cmssw:CMSSW_10_6_12
cmssw/cmssw:CMSSW_10_2_18

The other image I used :

cmssw/cc7:x86_64

This one is what is used on lxplus, which has cmsrel etc. But it is mounted to cvmfs and uses singularity. Which is something I am not even sure is possible to setup on CircleCI nodes.

Reply:
Also the commit ID attached to this PR is the same ID as in MELA/master. So I don't think the commit ID of Mela is the issue here. Mohit did update it, but the only way the 
`import mela`
works is if we linked the python bindings upon compilation which I was unable to do on CircleCi.